kontent-delivery/src/test/java/kentico/kontent/delivery/nestedmodels/Page.java

```diff
@@ -0,0 +1,43 @@
+package kentico.kontent.delivery.nestedmodels;
+
+import kentico.kontent.delivery.ContentItemMapping;
+import kentico.kontent.delivery.ElementMapping;
+import kentico.kontent.delivery.System;
+
+import java.util.List;
+
+@ContentItemMapping("page")
+public class Page {
+  @ElementMapping("title")
+  String title;
+
+  // Custom strongly type mapping to List
+  @ContentItemMapping("sections")
+  List<Section> sections;
```

Comment:
TODO:

* [ ] - extend tests for Map/All linked items/... as is is in [ListItem](https://github.com/Kentico/kontent-java-packages/blob/master/kontent-delivery/src/test/java/kentico/kontent/delivery/ArticleItem.java#L49-L74)